code/datums/uplink/gear loadout.dm

```diff
@@ -54,14 +54,14 @@
 /datum/uplink_item/item/gear_loadout/dominia_single
 	name = "Empire of Dominia Assets (Single)"
 	path = /obj/structure/closet/crate/secure/gear_loadout/dominia/single
-	item_cost = DEFAULT_TELECRYSTAL_AMOUNT
+	item_cost = 15
 
 /datum/uplink_item/item/gear_loadout/cowboys
 	name = "Frontier Cowboys (Group)"
 	path = /obj/structure/closet/crate/secure/gear_loadout/ram_ranch
-	item_cost = DEFAULT_TELECRYSTAL_AMOUNT*2
+	item_cost = 50
```

Comment:
DEFAULT_TELECRYSTAL_AMOUNT*2 is the same as 50, so if you want to make this cheaper like the others, find a fitting new price.

Reply:
I decided to leave it at the same price (as it offers extra value like the other group crates), but redid it as 50 for the sake of uniformity (and because if default TCs are changed, it would end up being the only item to be altered by that).

Reply:
Further explanation on that can be found on the thread I made, if you're still curious.